common/views/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -169,6 +185,35 @@ const SegmentedControl: FunctionComponent<Props> = ({
   const { isEnhanced } = useContext(AppContext);
   const [isActive, setIsActive] = useState(false);
 
+  function onClick(
+    e: MouseEvent<HTMLAnchorElement>,
+    item: SegmentedControlItem
+  ): boolean | undefined {
+    const url = e.currentTarget.href;
+    const isHash = url.startsWith('#');
+
+    trackGaEvent({
+      category: 'SegmentedControl',
+      action: 'select segment',
+      label: item.text,
+    });
+
+    setActiveId && setActiveId(item.id);
+    isEnhanced && setIsActive(false);
```

Comment:
This is the only difference between the two branches; imo pulling them into a single function makes this clearer.